### PR TITLE
Use adbTimeout to prevent timeout on apk install on device (issue square#176 )

### PR DESCRIPTION
**Pull request by intelradoux**
Got timeout when 10 devices are connected to a hub linked to my computer. This is a fix.

### CODE REVIEW

**Comment by edenman** [comment]
Can you revert all the import changes please?

File: spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java [hunk 1]
```diff
@@ -163,6 +166,9 @@ public DeviceResult run(AndroidDebugBridge adb) {
     final DeviceDetails deviceDetails = DeviceDetails.createForDevice(device);
     result.setDeviceDetails(deviceDetails);
     logDebug(debug, "[%s] setDeviceDetails %s", serial, deviceDetails);
+    
+    // Set timeout
```
Comment (JakeWharton) [inline spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java]: This comment is not useful

File: spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java [hunk 2]
```diff
@@ -26,7 +26,6 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-
```
Comment (edenman) [inline spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java]: revert this: there should be an empty line between regular imports and static imports

**Comment by edenman** [comment]
Needs a rebase

**Pull request closed without merging**